.buildkite/commands/assemble-release-apk.sh

```diff
@@ -1,5 +1,9 @@
 #!/bin/bash -eu
 
+if .buildkite/commands/should-skip-job.sh --job-type build; then
```

Comment:
What do you think of using this:

```suggestion
if "$(dirname "${BASH_SOURCE[0]}")/should-skip-job.sh --job-type build; then
```

I like it because it makes the path relative. I doubt we'll ever want to move the files from where they are, but I like that we _could_, seems like a good practice to have.